src/main/kotlin/org/rust/lang/core/psi/ext/RsDocAndAttributeOwner.kt

```diff
@@ -156,6 +251,10 @@ class QueryAttributes(
     val langAttribute: String?
```

Comment:
Do we want to use this field after introducing `langAttributes` (I'm just not sure)?

Reply:
Yep, `langAttributes` is used only during stubs building. After cfg_attr evaluation there should exist only one lang item